### DISCUSSION TITLE
3rd party error

### DISCUSSION THREAD

**Discussion by ansufw** [pdfcpu/pdfcpu#688] [Q&A]
I got an error when I ran the test for my repo (https://github.com/aysf/gopdf) and here is the error:

```
    reorder_test.go:35: code=400, message=error merging splitted files: dict=annotDict required entry=Subtype missing
```

looks like there is missing in my code instead of a bug. could you please help me to figure out why this error come out?

thanks in advance

**Comment by hhrutter** [comment]
Sorry, I do not have the time to discuss 3rd party issues.

Please identify the pdfcpu usecase that raises this error and file an issue.
Always ensure your input files pass validation!

Thank you!
